manifests/charts/cluster-operator/templates/deployment.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: kurator-cluster-operator
+    control-plane: kurator-cluster-operator
```

Comment:
app is widely used, so i am confused why `control-plane` here

Reply:
Actually does not provide services, but it'd not a bad idea to use standard labels.